SettingsFunctions.R

```diff
@@ -17,9 +17,9 @@ createCohortDiagnosticsModuleSpecifications <- function(runInclusionStatistics =
   }
 
   specifications <- list(module = "CohortDiagnosticsModule",
-                         version = "0.0.2",
+                         version = "0.0.8",
                          remoteRepo = "github.com",
-                         remoteUsername = "ohdsi",
+                         remoteUsername = "anthonysena",
```

Comment:
This should be changed/rejected to use the next version and to retain the remoteUsername == 'ohdsi'